models/run_results.yml

```diff
@@ -1,6 +1,85 @@
 version: 2
 
 models:
+  - name: dbt_invocations
+    description: >
+      Attributes associated with each dbt invocation. Inserted at the end of each invocation.
+
+    columns:
+      - name: invocation_id
+        data_type: string
+        tests:
+          - not_null
```

Comment:
I'm not sure if you want tests in these models, but it is useful for users trying to understand the relationships

Reply:
Yeah it's definitely useful! the reason we didn't add tests here so far was that we don't want to ask our users to actively exclude these tests when they run `dbt test`. So unless we have a good solution for that for now I'd avoid adding it here